client/scss/components/_workflow-tasks.scss

```diff
@@ -1,30 +1,30 @@
 .workflow-tasks {
-  $task-width: 117px;
-  $task-height: 56px;
+  $task-width: 7.313rem;
```

Comment:
This seems off, should be `7.3125rem`. Could you make sure to remove all rounding you’ve done in this file? We don’t want this change to visually alter the appearance of the component.